### PR TITLE
🐛 added checking multisig address before rewriting #LSC-168

### PR DESCRIPTION
**Pull request by callmepak**

### CODE REVIEW

File: multisig-helper.py [hunk 1]
```diff
@@ -651,17 +664,21 @@ def generate_multisig_account(name, ledger, google_api_token, config_path):
     participants = sorted([x[0] for x in result["values"]])
     multisig_account_name = "_".join(participants) + "_multisig"
     if key_exists(multisig_account_name, ledger):
-        print("Multisig account with name '%s' already exists on your device! Remove it or rename" % multisig_account_name)
-        exit(1)
+        print("Multisig account with name '%s' already exists on your device! Remove it or rename" %
+              multisig_account_name)
+        # exit(1)
```
Comment (pr0n00gler) [inline multisig-helper.py:669]: Why is this line commented out? This change breaks the script a little bit

File: multisig-helper.py [hunk 2]
```diff
@@ -690,18 +707,27 @@ def generate_multisig_account(name, ledger, google_api_token, config_path):
...
+            'values': [[" ".join(create_multisig_account_command)]]
+        }
+        sheet.values().update(
+            spreadsheetId=config["spreadsheet_id"], range=GENERATE_MULTISIG_COMMAND_CELL,
+            body=body, valueInputOption="RAW").execute()
+
+        print("Done")
+
+    if result["values"][0][0] != json_result["address"]:
+        print("multisig address is not equal!")
```
Comment (pr0n00gler) [inline multisig-helper.py:729]: More clearer message about what happened? Something like "your locally generated multisig address is not equal to the one in the Google Sheets".

File: multisig-helper.py [hunk 3]
```diff
@@ -690,18 +707,27 @@ def generate_multisig_account(name, ledger, google_api_token, config_path):
...
-        'values': [[" ".join(create_multisig_account_command)]]
-    }
-    sheet.values().update(
-        spreadsheetId=config["spreadsheet_id"], range=GENERATE_MULTISIG_COMMAND_CELL,
-        body=body, valueInputOption="RAW").execute()
+    result = sheet.values().get(spreadsheetId=config["spreadsheet_id"],
+                                range=MULTISIG_ADDRESS_CELL).execute()
 
-    print("Done")
+    if not result["values"][0][0]:
```
Comment (pr0n00gler) [inline multisig-helper.py:714]: The script fails on this line when a multisig address cell in Google Sheets is empty:
`KeyError: 'values'`

**Review by mmikev — CHANGES_REQUESTED** [review]

**Review by mmikev — APPROVED** [review]

File: multisig-helper.py [hunk 4]
```diff
@@ -711,7 +711,7 @@ def generate_multisig_account(name, ledger, google_api_token, config_path):
     result = sheet.values().get(spreadsheetId=config["spreadsheet_id"],
                                 range=MULTISIG_ADDRESS_CELL).execute()
 
-    if not result["values"][0][0]:
+    if result.get("values") is None:
```
Comment (oopcode) [inline multisig-helper.py:714]: `if not result.get("values")` is more idiomatic

**Pull request merged**